lightwood/data/encoded_ds.py

```diff
@@ -72,6 +72,8 @@ def __getitem__(self, idx: int) -> Tuple[torch.Tensor, torch.Tensor]:
 
                 data = self.data_frame[cols].iloc[idx].tolist()
                 encoded_tensor = self.encoders[col].encode(data, **kwargs)[0]
+                if torch.isnan(encoded_tensor).any() or torch.isinf(encoded_tensor).any():
```

Comment:
just to make sure, there is no weird buggy edge cases on nan/infs? I always shiver when I think of isnan() and isinf() calls : /

Reply:
There isn't because this is torch not python numbers and/or numpy numbers